Presentation.Web/app/shared/singleSelect/singleSelectSelect2.directive.ts

```diff
@@ -11,6 +11,19 @@
                 appendurl: "@",
                 disabled: "=ngDisabled",
                 required: "@"
+            },
+            link(scope) {
+
+                scope.optionDescription = null;
+
+                scope.$watch('model.selectedElement',
+                    value => {
+                        if (value) {
+                            scope.optionDescription = value.optionalObjectContext.description;
```

Comment:
if there is no Optional object context then this will fail

Use null propagation `value.optionalObjectContext?description`

Reply:
actually it could all be rewritten into a single statement:
`scope.optionDescription = value?.optionalObjectContext?.description;`

Reply:
so no if/else